library/network_connections.py

```diff
@@ -348,7 +349,10 @@ def ifcfg_create(
                 if (connection["infiniband"]["transport_mode"] == "connected")
                 else "no"
             )
-            if connection["infiniband"]["p_key"] != -1:
+            if (
+                connection["infiniband"]["p_key"]
+                != ArgValidator_DictInfiniband.DEFAULT_PKEY_VAL
```

Comment:
```suggestion
                != ArgValidator_DictInfiniband.DEFAULT_PKEY
```

The `_VAL` suffix is not needed. IMHO it is generally clear that a constant represents a value (assuming that VAL is the abbreviation of value).

Reply:
Changed.